src/main/java/lesson6/Car.java

```diff
@@ -0,0 +1,30 @@
+package lesson6;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@AllArgsConstructor
+public class Car {
+
+    @Getter
+    private double tankCapacity;
+    @Getter
+    private double fuelPerHundredKm;
+    @Getter
+    private long lastInspectionTime;
+
+    public boolean canRide() {
+        return tankCapacity > 0.0;
+    }
+
+    public boolean isOperational() {
+        long twoYearsInMillis = 2 * 31536000000L;
```

Comment:
Why did you convert it to millis? 😄 
Even if you do so, you could have used already multiplied number `63072000000L`